src/Authenticator/AuthenticatorCollection.php

```diff
@@ -65,12 +65,12 @@ protected function _create($className, $alias, $config)
     /**
      * Resolves authenticator class name.
      *
-     * @param string $class Class name to be resolbed.
-     * @return string
+     * @param string $class Class name to be resolved.
+     * @return string|null
      */
     protected function _resolveClassName($class)
     {
-        return App::className($class, 'Authenticator', 'Authenticator');
+        return (string)App::className($class, 'Authenticator', 'Authenticator') ?: null;
```

Comment:
I doubt it causes any noticeable degradation in performance but I still don't like adding it. We could stay on level 5 and ignore this particular error instead.